src/server/api/controllers/private-channel.controller.js

```diff
@@ -0,0 +1,17 @@
+const knex = require('../../config/db');
+// const Error = require('../lib/utils/http-error');
+// const moment = require('moment-timezone');
+
+const checkPrivateChannel = async (userId1, userId2) => {
+  const result = await knex('channel_members as a')
+    .select('a.fk_channel_id', 'a.fk_user_id')
+    .join('channel_members as b', 'b.fk_channel_id', 'a.fk_channel_id')
+    .groupBy('a.id')
+    .havingIn('a.fk_user_id', [userId1, userId2]);
```

Comment:
Awesome, well done

Reply:
Hi Wael. I check that in database. It is not return channel, where 2 or more users. It is return the same information about channels. If you want select channel, where was 2 or more users, this work in database
select fk_channel_id from channel_members
where fk_user_id in (1,3)
group by fk_channel_id
having count(*) = 2;
you can change user_id and add more and change count('*) 
In knex you should think how add userIds and count
What do you think?